### ISSUE TITLE
requirements

### ISSUE THREAD

**Issue by memphizz** [daniel-munro/imageCCA#2]
Hi,
Could you please share your requirements/dependencies for the CAE code?

Thanks!

**Comment by daniel-munro** [comment]
Hello, I've tried this code with recent versions of the Python packages used in the scripts, but unfortunately had many issues, even with modifications. I got it to run, though perhaps without utilizing GPU, using keras=1.2.2 and theano=1.0.1 (python 2.7). Note that it must use theano for the backend rather than tensorflow.

If you want to use this CAE model, I think your best bet is to try using up-to-date packages and modifying things as needed until it works. I'd like to do that myself when I get a chance, but I don't know when that will be.

**Comment by memphizz** [comment]
Thanks for the response. I ended up rewriting it using TF 2.6. The mse loss goes to 0.027 for the training set but the images are still blurry. What was the lowest mse loss you were getting? Can you share a few sample pairs from your test set?

**Comment by daniel-munro** [comment]
We've tried looking back at the loss logs, but values ranged from 0.009 to the thousands in different logs, and it's not clear which ones represent MSE loss, log MSE loss, total batch loss, etc. Fig 6 of the paper gives one example of how much blurriness to expect in the "output" image, but I don't have more examples.
